src/Interpreters/ApplyWithSubqueryVisitor.h

```diff
@@ -15,21 +16,26 @@ struct ASTTableExpression;
 class ApplyWithSubqueryVisitor
 {
 public:
+    ApplyWithSubqueryVisitor() = delete;
```

Comment:
(The presence of an explicitly defined constructor, e.g. the one in l. 20, implicitly deletes all other constructors. Basically, you can delete l. 19.)